package.json

```diff
@@ -13,8 +13,7 @@
     "url": "https://github.com/ubolonton/js-csp/issues"
   },
   "dependencies": {
-    "babel-runtime": "^6.22.0",
-    "lodash": "^4.17.4"
+    "babel-runtime": "^6.22.0"
```

Comment:
As we discuss, we can re-add lodash and use its specific functions for our needs to prevent large bundle size